presto-main/src/main/java/io/prestosql/sql/gen/SqlTypeBytecodeExpression.java

```diff
@@ -68,7 +78,7 @@ public BytecodeNode getBytecode(MethodGenerationContext generationContext)
     @Override
     protected String formatOneLine()
     {
-        return type.getTypeSignature().toString();
+        return generateName(type);
```

Comment:
I think we want to keep the original here, since this is only used for debugging, not code generation.

Reply:
Reverted the change.